rhel7/profiles/ccc.profile

```diff
@@ -0,0 +1,271 @@
+#    - audit_rules_unsuccessful_file_modification_chmod
+#    - audit_rules_unsuccessful_file_modification_fchmod
+#    - audit_rules_unsuccessful_file_modification_fchmodat
+#    - audit_rules_unsuccessful_file_modification_setxattr
+#    - audit_rules_unsuccessful_file_modification_lsetxattr
+#    - audit_rules_unsuccessful_file_modification_fsetxattr
+#    - audit_rules_unsuccessful_file_modification_removexattr
+#    - audit_rules_unsuccessful_file_modification_lremovexattr
+#    - audit_rules_unsuccessful_file_modification_fremovexattr
+#    - audit_rules_unsuccessful_file_modification_lchown
+#    - audit_rules_unsuccessful_file_modification_fchown
+#    - audit_rules_unsuccessful_file_modification_chown
+#    - audit_rules_unsuccessful_file_modification_fchownat
+#    - audit_rules_etc_passwd_openat
+#    - audit_rules_etc_passwd_open_by_handle_at
+#    - audit_rules_etc_passwd_open
+#    - audit_rules_etc_shadow_openat
+#    - audit_rules_etc_shadow_open_by_handle_at
+#    - audit_rules_etc_shadow_open
+#    - audit_rules_etc_group_openat
+#    - audit_rules_etc_group_open_by_handle_at
+#    - audit_rules_etc_group_open
+#    - audit_rules_etc_gshadow_openat
+#    - audit_rules_etc_gshadow_open_by_handle_at
+#    - audit_rules_etc_gshadow_open
+#    - audit_rules_privileged_commands_unix_chkpwd
+#    - audit_rules_privileged_commands_userhelper
+#    - audit_rules_privileged_commands_usernetctl
+#    - audit_rules_execution_seunshare
+#    - audit_rules_privileged_commands_mount
+#    - audit_rules_privileged_commands_newgrp
+#    - audit_rules_privileged_commands_newuidmap
+#    - audit_rules_privileged_commands_gpasswd
+#    - audit_rules_privileged_commands_newgidmap
+#    - audit_rules_privileged_commands_umount
+#    - audit_rules_privileged_commands_passwd
+#    - audit_rules_privileged_commands_crontab
+#    - audit_rules_mac_modification
+#    - audit_rules_session_events
+#    - audit_rules_privileged_commands_at
+    - directory_access_var_log_audit
+#    - audit_rules_kernel_module_loading_insmod
+#    - audit_rules_kernel_module_loading_rmmod
+#    - audit_rules_kernel_module_loading_modprobe
+#    - audit_rules_kernel_module_loading_init
+#    - audit_rules_kernel_module_loading_finit
+#    - audit_rules_kernel_module_loading_delete
+
+    # Configuration (all are defaults)
+#    - var_auditd_data_retention=incremental_async  # typo? var_auditd_flush used instead
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+
+    # audispd plugins
+    - auditd_audispd_syslog_plugin_activated
+
+    ### Services
+
+    # sshd
+    - sshd_disable_root_login
+    - sshd_enable_strictmodes
+    - disable_host_auth
+    - sshd_disable_user_known_hosts
+    - sshd_disable_rhosts
+    - sshd_disable_empty_passwords
+    - sshd_disable_kerb_auth
+    - sshd_disable_gssapi_auth
+    - sshd_idle_timeout_value=10_minutes
+#    - sshd_set_idle_timeout  # duplicate, see below
```

Comment:
I think you can drop this comment.
Unless you would like to keep track of this for some reason.